app/controllers/cart/cart_items_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Cart::CartItemsController < ApplicationController
+  before_action :set_cart_item, only: %i[update]
+
+  def create
+    product = Product.purchasable.find(cart_item_params[:product_id])
+    current_cart.add_cart_item(product, cart_item_params[:quantity])
+    redirect_to cart_url, notice: '商品をカートに追加しました'
+  end
+
+  def update
+    if cart_item_params[:quantity].to_i.zero?
+      @cart_item.destroy!
+      redirect_to cart_url, notice: '商品をカートから削除しました'
+    else
+      @cart_item.update!(quantity: cart_item_params[:quantity].to_i)
+      redirect_to cart_url, notice: '商品個数を更新しました'
+    end
```

Comment:
多少好みもありますが、繰り返し使うなら変数に入れるのがよいと思います
```suggestion
    quantity = cart_item_params[:quantity].to_i
    if quantity == 0 # .zero? が分かりやすい気はしないので...(rubocop のこの指摘は無効にしてしまえばよいかと)
      @cart_item.destroy!
      redirect_to cart_url, notice: '商品をカートから削除しました'
    else
      @cart_item.update!(quantity:)
      redirect_to cart_url, notice: '商品個数を更新しました'
    end
```

Reply:
[3947015](https://github.com/kk0000-kk/sakura-market/pull/25/commits/39470150c71a91ac5a1b3ad5f2e65b521b2a71f1)